convert2rhel/unit_tests/logger_test.py

```diff
@@ -32,7 +33,8 @@ def test_logger_handlers(monkeypatch, tmpdir, caplog, read_std, is_py2, global_t
     # initializing the logger first
     log_fname = "convert2rhel.log"
     global_tool_opts.debug = True  # debug entries > stdout if True
-    logger_module.setup_logger_handler(log_name=log_fname, log_dir=str(tmpdir))
+    logger_module.setup_logger_handler()
```

Comment:
Can we get rid of the `caplog` and `is_py2` fixtures in this test? They seem to not be used.